spec/features/admin_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+describe 'admin', type: :feature do
+
+  let(:admin) { User.create(username: 'admin', password: 'password',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.